### PR TITLE
CHANGELOG: add 3.6, highlight completion commands

### PR DESCRIPTION
**Pull request by avorima**

### CODE REVIEW

File: CHANGELOG-3.6.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+
+
+Previous change logs can be found at [CHANGELOG-3.5](https://github.com/etcd-io/etcd/blob/main/CHANGELOG-3.5.md).
+
+<hr>
+
+## v3.6.0 (TBD)
+
+See [code changes](https://github.com/etcd-io/etcd/compare/v3.5.0...v3.6.0) and [v3.5 upgrade guide](https://etcd.io/docs/latest/upgrades/upgrade_3_5/) for any breaking changes.
```
Comment (lilic) [inline CHANGELOG-3.6.md:9]: Since we don't have the upgrade guide yet for 3.6, lets skip this maybe?

File: CHANGELOG-3.6.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+### etcdctl v3
+
+- Add command to generate shell completion. ([PR#13133](https://github.com/etcd-io/etcd/pull/13133)).
```
Comment (lilic) [inline CHANGELOG-3.6.md:13]: Nit: I believe this is more like in style to how the previous changelog was formatted, same with the other one: 

```suggestion
- Add command to generate [shell completion](https://github.com/etcd-io/etcd/pull/13133).
```

**Review by lilic — COMMENTED** [review]
Nice, thank you! Few suggestions.

Would it make sense to also add this to our docs for more visibility or is this enough?

File: CHANGELOG-3.6.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+
+
+Previous change logs can be found at [CHANGELOG-3.5](https://github.com/etcd-io/etcd/blob/main/CHANGELOG-3.5.md).
```
Comment (spzala) [inline CHANGELOG-3.6.md:3]: lgtm to start with the new changelog. We should be adding recommendation on versions as usual but it's OK to do it later.

File: CHANGELOG-3.6.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+## v3.6.0 (TBD)
+
+See [code changes](https://github.com/etcd-io/etcd/compare/v3.5.0...v3.6.0).
+
+### etcdctl v3
+
+- Add command to generate [shell completion](https://github.com/etcd-io/etcd/pull/13133).
+
+### etcdutl v3
```
Comment (spzala) [inline CHANGELOG-3.6.md:15]: Before merging this, let's wait on https://github.com/etcd-io/etcd/pull/13142 to merge.

File: CHANGELOG-3.6.md (hunk 3 above)
Comment (avorima) [reply CHANGELOG-3.6.md:3]: Should I add 3.2.28, 3.3.18 and 3.4.2 like in the other CHANGELOGs?

File: CHANGELOG-3.6.md (hunk 2 above)
Comment (avorima) [reply CHANGELOG-3.6.md]: Done

File: CHANGELOG-3.6.md (hunk 3 above)
Comment (spzala) [reply CHANGELOG-3.6.md:3]: Thanks for the quick update @avorima. About recommended versions we should list `3.3.18+, 3.4.2+ and 3.5.0+` but please hold on it for now - cc @gyuho @hexfusion @ptabor I will have it discussed in monthly meeting tomorrow if needed to wait by then. Thanks!

File: CHANGELOG-3.6.md (hunk 4 above)
Comment (ptabor) [reply CHANGELOG-3.6.md:15]: #13142 Merged/

File: CHANGELOG-3.6.md (hunk 3 above)
Comment (spzala) [reply CHANGELOG-3.6.md:3]: @avorima we had it discussed in the community meeting today and we are waiting on few feedback, so we will add versions detail separately.

**Comment by spzala** [comment]
Merging this to have CHANGELOG file in place for other changes. The updates on recommendation needs to be done at other places as well, we can do all together separately.

**Pull request merged**